db/index.py

```diff
@@ -189,6 +189,19 @@ def get_push_key():
     # [END push_key]
     print(post_id)
 
+def run_transaction():
+    # [START transaction]
+    def increment_votes(current_value):
+        return current_value + 1 if current_value else 1
+
+    upvotes_ref = db.reference('server/saving-data/fireblog/posts/-JRHTHaIs-jNPLXOQivY/upvotes')
```

Comment:
`run_transaction()` is a method defined on the `db.Reference` object. This is similar to the corresponding Node.js and Java APIs: https://firebase.google.com/docs/database/admin/save-data#section-transactions

Note that the path comes from the existing code samples.